src/tooltip/stories.tsx

```diff
@@ -16,7 +16,11 @@ const meta = {
 export default meta;
 type Story = StoryObj<typeof meta>;
 
-export const Default: Story = {};
+export const Default: Story = {
+    args: {
+        children: <TooltipTriggerButton slot="clear" />,
```

Comment:
Don't think this needs `slot="clear"` here?

Reply:
...saves us doing it in the portal...?